### ISSUE TITLE
DropPrimaryKey doesn't work on Oracle

### ISSUE THREAD

**Issue by cdprete** [liquibase/liquibase#1863]
Liquibase version: 3.10.3
DBMS vendor & version: Oracle 11 XE

The `dropPrimaryKey`  change fails with the following error on Oracle:

```
Error: ORA-02429: cannot drop index used for enforcement of unique/primary key
 [Failed SQL: (2429) ALTER TABLE MY_TABLE DROP CONSTRAINT MY_CONSTRAINT_NAME DROP INDEX]
```

**Comment by molivasdat** [comment]
Hi @cdprete Thanks for sharing this issue.  Can you share an example changeset that fails?  Are there other constraints on the table?

**Comment by cdprete** [comment]
HI @molivasdat 
Here the example:
```xml
<changeSet author="cdprete" id="drop-pk"> 
    <dropPrimaryKey constraintName="MY_CONSTRAINT_NAME" tableName="MY_TABLE"/>
 </changeSet>
 ```

I honestly don't remember now if there are other constraints on the table, but this should not matter since for Postgres it works fine :)

**Comment by FBurguer** [comment]
Looks like the issue is solved for liquibase 4.11-4.12
First Changeset:
```
<changeSet id="1" author="your.name" labels="example-label" context="example-context">
        <comment>example-comment</comment>
        <createTable tableName="person">
            <column name="id" type="int" autoIncrement="true">
                <constraints primaryKey="true" nullable="false"/>
            </column>
            <column name="name" type="varchar(50)">
                <constraints nullable="false"/>
            </column>
            <column name="address1" type="varchar(50)"/>
            <column name="address2" type="varchar(50)"/>
            <column name="city" type="varchar(30)"/>
        </createTable>
    </changeSet>
```
Second Changeset:
```
    <changeSet author="your.name" id="2"> 
        <dropPrimaryKey constraintName="PK_PERSON" tableName="PERSON"/>
    </changeSet>
```

And it looks like `PK_PERSON`  has been dropped

Test Environment:
OS: Windows 10
JAVA 17
Docker Image: gvenzl/oracle-xe  version 21.3.0
